src/test/initialize.ts

```diff
@@ -134,10 +134,10 @@ async function closeWindowsInternal() {
 function isANotebookOpen() {
     /* eslint-disable */
     if (
-        Array.isArray((vscode as any).notebook.visibleNotebookEditors) &&
-        (vscode as any).notebook.visibleNotebookEditors.length
+        Array.isArray((vscode as any).window.visibleNotebookEditors) &&
```

Comment:
Do we have a reason for why we did this?

Reply:
I think it was due to some breaking changes in vscode and I wanted to future proof it.
War should be able to remove that now

Reply:
Yup, now remember. We couldn't use proposed  api in stable (at one point, until we enabled the experiment and supported it) and this block of code always ran in stable as well. It has to compile and run in stable.

Reply:
Yeah, this change was added by Rich.